IoT_master/tools/camera_initialize.py

```diff
@@ -0,0 +1,37 @@
+import cv2
+
+class Camera:
```

Comment:
I’m not sure whether `Camera` speaks for itself. Maybe `ThermalCamera` would be more meaningful